.github/workflows/gfi_automate_assignment.yml

```diff
@@ -0,0 +1,268 @@
+name: GFI assigned task automation
+
+on:
+  issues:
+    types: [assigned, unassigned]
+
+jobs:
+  issue_assigned:
+    if: github.event.action == 'assigned'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Move issue to Assigned column
+        uses: actions/github-script@v4
```

Comment:
Script is not small one, maybe it is better to wrap it into the custom JS action? Then we will be able to use it in different openvino repositories
https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-javascript-action